crates/primitives/src/alloy_compat.rs

```diff
@@ -5,7 +5,7 @@ use crate::{
     Transaction, TransactionSigned, TransactionSignedEcRecovered, TransactionSignedNoHash, TxType,
 };
 use alloc::{string::ToString, vec::Vec};
-use alloy_consensus::{TxEip1559, TxEip2930, TxEip4844, TxLegacy};
+use alloy_consensus::{Transaction as _, TxEip1559, TxEip2930, TxEip4844, TxLegacy};
```

Comment:
@mattsse we need this because `Transaction` name conflicts with another imported type. We can put `as AlloyTransaction` but this seems equivalent there no?

Reply:
gotcha, that makes sense